Authentication/middleware/fetchuser.js

```diff
@@ -0,0 +1,20 @@
+const jwt = require('jsonwebtoken');
```

Comment:
why does this need to be in a separate file? cant it be put inside checkAuthentication.js

Reply:
checkAuthentication and fetchuser file serve different functionalities so I think a different file is fine